unittest/python/crocoddyl_legacy/integrated_action.py

```diff
@@ -1,10 +1,11 @@
 import numpy as np
+import crocoddyl
 
 
 class IntegratedActionModelEuler:
```

Comment:
You need to revert the change of folder of each file inside the `unittest/python/crocoddyl`.
You should not code anything there.

Reply:
done

Reply:
done